GoScrumApi/src/main/java/com/goscrum/GoScrumApi/entity/Room.java

```diff
@@ -0,0 +1,84 @@
+package com.GoScrum.GoScrumApi.entity;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Size;
+import lombok.*;
+
+import java.nio.charset.Charset;
+import java.util.Random;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+
+// Database level
+@Entity
+@Table(name = "rooms", uniqueConstraints = {@UniqueConstraint(columnNames = {"slug"})})
```

Comment:
> @Data is a convenient shortcut annotation that bundles the features of [@ToString](https://projectlombok.org/features/ToString), [@EqualsAndHashCode](https://projectlombok.org/features/EqualsAndHashCode), [@Getter / @Setter](https://projectlombok.org/features/GetterSetter) and [@RequiredArgsConstructor](https://projectlombok.org/features/constructor)
https://projectlombok.org/features/Data

Reply:
Thêm cái @Builder hoặc @SuperBuilder nữa, để sử dụng builder pattern out of the box